manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: dns
+  namespace: openshift-dns-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: openshift-dns.rules
+      rules:
+      - alert: CoreDNSPanicCount
+        expr: coredns_panic_count_total > 0
+        for: 5m
```

Comment:
Why 5m?  Would it make more sense to report a panic immediately?

Reply:
Meant to remove this line, good catch! You're correct, there's no reason to wait on this one.